lib/galaxy/config/sample/galaxy.yml.sample

```diff
@@ -1796,6 +1796,22 @@ galaxy:
   # persistent_communication_rooms is a comma-separated list of rooms
   # that should be always available.
   #persistent_communication_rooms: null
+  
```

Comment:
I believe these new config properties should be added to `webapps/galaxy/config_schema.yml`, then  run `make config rebuild` to rebuild all sample yaml and rst files.